### DISCUSSION TITLE
论文中出现的中文偏僻字不显示

### DISCUSSION THREAD

**Discussion by fantasyEthan** [nju-lug/NJUThesis#138] [Q&A]
<img width="310" alt="image" src="https://user-images.githubusercontent.com/54493995/166432824-426a2ba9-42d7-4987-b18c-29dfde7965f0.png">
<img width="402" alt="image" src="https://user-images.githubusercontent.com/54493995/166432861-78b6a8ac-3f0b-4d3f-be22-21ba3e02e4d9.png">
如图，第一个作者的最后一个字似乎不能正常显示

**Comment by atxy-blip** [comment]
如果没猜错的话你在用 <https://tex.nju.edu.cn/>，默认使用的 fandol 字体确实有缺字的问题。

解决办法也很简单，文档类选项里指定一下用 Windows 字库：

```TeX
\documentclass[
    ...,
    latin-font = win,
    cjk-font = win
    ]{njuthesis}

...
```

**Reply by fantasyEthan** [reply]
嗯嗯，谢谢作者的回复，已经按照上面的方法解决了
